### PR TITLE
feat(owner): return subOrgs from org owner endpoint

### PR DESCRIPTION
**Pull request by dwojno**
I've added the subOrgs to the get orgs by owner endpoint.

Also added some fixes regarding not removing subOrgs while ENS sync

### CODE REVIEW

File: src/claim/claim.service.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export class ClaimService {
     const res = await this.dgraph.query(
       `
       query all($did: string) {
-        claim(func: type(Claim) @filter(eq(issuedToken, $did) OR eq(requester, $did)) {
+        claim(func: type(Claim)) @filter(eq(claimIssuer, $did) OR eq(requester, $did)) {
```
Comment (jrhender) [inline src/claim/claim.service.ts:148]: 👍  Nice

File: src/owner/owner.service.ts [hunk 2]
```diff
@@ -37,15 +40,27 @@ export class OwnerService {
    * @param type
    * @private
    */
-  private async getTypeByOwner(owner: string, type: string) {
+  private async getTypeByOwner(owner: string, type: 'App' | 'Org' | 'Role') {
```
Comment (jrhender) [inline src/owner/owner.service.ts:43]: Nice 👍

**Review by jrhender — APPROVED** [review]
Looks good. One minor suggestion to consider.

File: src/ENS/ens.service.ts [hunk 3]
```diff
@@ -20,6 +20,8 @@ import { namehash } from '../ethers/utils';
 import { OwnerService } from '../owner/owner.service';
 import chunk from 'lodash.chunk';
 
+const emptyAddress = '0x0000000000000000000000000000000000000000';
```
[the remark this answers is not in the document]
Comment (dwojno) [reply src/ENS/ens.service.ts]: 👍

**Pull request merged**